app/views/api/integrations/apicast/shared/_mapping_rules.html.slim

```diff
@@ -9,26 +9,53 @@
           - if @service.using_proxy_pro?
             th Redirect
           th title="increment" +
-          th colspan="2"
+          th
             ' Metric or Method
             | (
             = link_to 'Define', admin_service_metrics_path(@service), title: 'Define Methods & Metrics for this service'
             | )
+          th colspan="2"
+            ' Last?
 
-      tbody
+      tbody#sortable.ui-sortable
         - metrics = options_from_collection_for_select(@service.metrics, 'id', 'name')
 
-        - @proxy.proxy_rules.each do |rule|
-          tr
-            = render 'api/integrations/apicast/shared/mapping_rule', form: f, rule: rule, metrics: metrics
-        tr#new-proxy-rule-template style="display:none;"
+        - @proxy.proxy_rules.ordered.each_with_index do |rule, i|
+          tr.ui-sortable-item
+            = render 'api/integrations/apicast/shared/mapping_rule', form: f, rule: rule, metrics: metrics, index: (i + 1)
+        tr#new-proxy-rule-template.ui-sortable-item style="display:none;"
           = render 'api/integrations/apicast/shared/mapping_rule', form: f, rule: @proxy.proxy_rules.new(delta: 1), disabled: true, metrics: metrics
       tfoot
         tr
-          th colspan=(@service.using_proxy_pro? ? '6' : '5')
+          th colspan=(@service.using_proxy_pro? ? '7' : '6')
             = link_to "Add Mapping Rule", '#add-proxy-rule', class: 'action add', id: 'add-proxy-rule'
 
 javascript:
-  jQuery('#proxy-rules .metric select[data-selected]').each(function(){
+  jQuery('#proxy-rules .metric select[data-selected]').each(function () {
     jQuery(this).val(jQuery(this).data('selected'))
   })
+  jQuery(function () {
```

Comment:
Is this really needed? As I understand, on drag (start) it adds 4px width on each td where the element was dragged.

Reply:
it happens here, it's more than 4px: https://github.com/3scale/porta/pull/530/files#diff-2eeed0d10ff9fe7aedaf9123ec70b297R39

Reply:
If you do not add this code, will it be ugly? Can it be done in pure CSS?

Reply:
Sadly nope, this trick is needed because of jqueryui#sortable lib and the use of tables.